### ISSUE TITLE
value/magnitude of DV_QUANTITY  in AQL returns string instead of number

### ISSUE THREAD

**Issue by sidharthramesh** [ehrbase/ehrbase#478]
## Configuration information

- EHRbase version: 0.14.0
- openEHR_SDK version: 29b57aa
- Archie version: 738152e
- PostgreSQL version: PostgreSQL 10.12 on x86_64-pc-linux-musl, compiled by gcc (Alpine 9.2.0) 9.2.0, 64-bit
- Java Runtime version: Oracle Corporation 11.0.9.1+1
- Operating System version: Linux amd64 4.19.128-microsoft-standard

## Steps to reproduce ## 

POST http://localhost:8080/ehrbase/rest/openehr/v1/query/aql
```json
{
    "q": "select o/data[at0001]/events[at0006]/data[at0003]/items[at0004]/value from OBSERVATION o [openEHR-EHR-OBSERVATION.blood_pressure.v2]"
}
```

Returns: 
```json
{
"rows":[
     [
      {
        "units": "mm[Hg]",
        "magnitude": 594.0
      }
    ]
  ]
}
```
Magnitude is number here. However, 
POST http://localhost:8080/ehrbase/rest/openehr/v1/query/aql
```json
{
    "q": "select o/data[at0001]/events[at0006]/data[at0003]/items[at0004]/value/magnitude from OBSERVATION o [openEHR-EHR-OBSERVATION.blood_pressure.v2]"
}
```

Returns
```
{
"rows": [
    [
      "594"
    ],
  ]
}
```
Magnitude is a string here.

## Expected result (Acceptance Criteria) ##
Querying value/quantity should also be a number.

## Definition of Done ##

- [ ] The defect is checked by an unit or an integration test (Robot)
- [ ] Merge Request approved
- [ ] Unit tests passed
- [ ] Build without errors
- [ ] Release notes prepared
- [ ] No additional runtime warnings

**Comment by chevalleyc** [comment]
Hi @sidharthramesh . I think this has been fixed in the current code base (0.16). Could you try it out? Thanks

**Comment by sidharthramesh** [comment]
Yup. Fixed in the current version! Thank you!
